packages/components/src/components/Table/TableContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Card } from '../Card/Card';
+
+export const Container = styled.div`
+    flex-direction: column;
+`;
+
+interface TableContainerProps {
+    children: ReactNode;
+}
+
+export const TableContainer = ({ children }: TableContainerProps) => (
```

Comment:
I'm thinking the other components could be exported as part of the Table component, i.e. Table.Row = TableRow and so on. Then you only have to import the main Table component. What do you think?

Reply:
I agree and I suggest following naming:

```
Table 
Table.Row 
Table.Cell 
Table.HeaderRow 
Table.HeaderCell